lib/votes/deleteVote.ts

```
@@ -0,0 +1,29 @@
+import { Vote } from '@prisma/client';
+import { hasAccessToSpace } from 'lib/users/hasAccessToSpace';
+import { DataNotFoundError } from 'lib/utilities/errors';
+import { prisma } from 'db';
+import { getVote } from './getVote';
+
+export async function deleteVote (id: string, userId: string): Promise<Vote | null> {
+  const vote: Vote | null = await getVote(id);
+
+  if (!vote) {
+    throw new DataNotFoundError(`There is no vote with id: ${id}`);
```

can we make this silently fail. there's no reason to throw an error if something doesn't exist that i asked to delete. just log a warning